scripts/build_renewable_profiles.py

```diff
@@ -532,6 +543,9 @@ def get_hydro_capacity_annual_hydro_generation(
             ]
         )
 
+        n_weights_initial = len(ds.weight)
+        n_lost_weights = pd.isnull(ds.weight).sum()
```

Comment:
Can we move that to line 572?

Reply:
Yes, and we can also simplify a bit a test on a share of lost buses